src/content/en/updates/2018/07/css-scroll-snap.md

```diff
@@ -0,0 +1,365 @@
+
+Chrome, Opera and Safari are shipping the latest specifications with the other
+major browser vendors planning to follow along in the near future
+([Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1231777),
+[Edge bug](https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18394960/)).
+
+This means you'll find several tutorials on the web which discuss the old syntax
+which is still currently implemented by Edge and Firefox.
+
+<figure>
+  <a
+    href="/web/updates/images/2018/07/css-scroll-snap/specification-history.png">
+    <img
+    src="/web/updates/images/2018/07/css-scroll-snap/specification-history.png"
+    alt="Major changes in scroll snap specification"> </a>
+  <figcaption>Major changes in scroll snap specification.</figcaption>
+</figure>
+
+
+## CSS Scroll Snap
+
+
+Scroll snapping is the act of adjusting the scroll offset of a scroll container
+to be at a preferred **snap position** once the scroll operation is finished.
+
+A scroll container may be opted into scroll snapping by using `scroll-snap-type`
+property. This tells the browser that it should consider snapping this scroll
+container to the snap positions produced by its descendents.  `scroll-snap-type`
+determines the axis on which scrolling occurs: `x`, `y`, or `both`, and the
+snapping strictness: `mandatory`, `proximity`. More on these later.
+
+A snap position can be produced by declaring a desired alignment on an element.
+This position is the scroll offset at which the nearest ancestor scroll
+container and the element are aligned as specified for the given axis. The
+following alignments are possible on each axis: `start`, `end`, `center`.
+
+A `start` alignment means that the scroll container snapport start edge should
+be flushed with the element snap area start edge. Similarly, the `end` and
+`center` alignments mean that the scroll container snapport end edge or center
+should be flushed with the element snap area end edge or center.
+
+[Snapport](https://drafts.csswg.org/css-scroll-snap/#scroll-padding) is the area
+of the scroll container to which the snap areas are aligned. By default it is
+the same as the visual viewport of the scroll container but it can be adjusted
+using `scroll-padding` property.
+
+
+
+<figure>
+    <video src="/web/updates/images/2018/07/css-scroll-snap/interactive-alignment.mp4"
+    autoplay loop muted playsinline alt="Alignment demo"> </video>
+  <figcaption>
+    Example of a various alignments on horizontal scrolling axis.
+ </figcaption>
+</figure>
+
+
+
+The following examples illustrate how these concepts can be used in practice.
+
+
+### Example - Horizontal gallery
+
+A common use case for scroll snapping is an  image carousel. For example, to
+create a horizontal image carousel that snaps to each image as you scroll, we
+can specify the scroll container to have a mandatory ``scroll-snap-type`` on the
+horizontal axis.  set each image to ``scroll-snap-align: center`` to ensure that
```

Comment:
double wrapped "scroll-snap-align: center" is wrapped in double back ticks, should be single , only once is necessary/used by devsite

Reply:
Done